src/interceptors/auth.interceptors.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  CallHandler,
+  ExecutionContext,
+  NestInterceptor,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { Observable } from 'rxjs';
+import { UserService } from 'src/user/user.service';
+
+export class AuthInterceptor implements NestInterceptor {
+  constructor(private readonly userService: UserService) {}
+
+  async intercept(
+    context: ExecutionContext,
+    handler: CallHandler,
+  ): Promise<Observable<any>> {
+    const req = context.switchToHttp().getRequest();
```

Comment:
you can add here in the top of the file 

```
  if (true)
  return handler.handle();
```
Just to be able to merge this PR withot breaking the app :)

Reply:
Notice that iIdefined : "req.userId = await this.userService.getUserBySpotifyId(spotifyUserId)", so for example if you want to create a playlist :  @Post('')
  async createPlaylist(@Body() playlistName: string, @Req() req: any) {
    return await this.playlistService.createPlaylist(req.userId, playlistName);
  }. And if i dont set the userId, a playlist cannot be created.